### PR TITLE
Add node density test for ocp-qe-perfscale-ci

### PR DESCRIPTION
**Pull request by mffiedler**
Initial add of ocp-qe-perfscale-ci node density test with parameters tuned down to ensure the test is working before running at full scale.

### CODE REVIEW

File: ci-operator/step-registry/openshift-qe/cluster-density/openshift-qe-cluster-density-commands.sh [hunk 1]
```diff
@@ -13,4 +13,4 @@ python --version
 pushd /tmp
 git clone https://github.com/cloud-bulldozer/e2e-benchmarking
 pushd e2e-benchmarking/workloads/kube-burner
-./run_nodedensity_test_fromgit.sh
+./run_clusterdensity_test_fromgit.sh
```
Comment (paigerube14) [inline ci-operator/step-registry/openshift-qe/cluster-density/openshift-qe-cluster-density-commands.sh:16]: do we need to add in an export of the JOB_ITERATIONS variable here similar to the node density test? Think 1000 is the default and that would be too many for a 2 node cluster

Comment (mffiedler) [reply ci-operator/step-registry/openshift-qe/cluster-density/openshift-qe-cluster-density-commands.sh:16]: We do not need JOB_ITERATIONS for node_density - just the number of nodes and the node density.   I'd like to run with these numbers in productions initially and change as needed.

**Comment by mffiedler** [comment]
@paigerube14 @skordas @rpattath @kedark3 Tests finally passed.  I'd like to run with this lower density in production a few times and then tweak it as needed.   Can some please add a /lgtm ?

**Review by kedark3 — APPROVED** [review]
/lgtm

**Pull request merged**